src/test/scala/nl.knaw.dans.easy.multideposit/parser/MultiDepositParserSpec.scala

```diff
@@ -331,4 +331,13 @@ class MultiDepositParserSpec extends TestSupportFixture with DepositTestObjects
       case Success(_) =>
     }
   }
+
+  it should "fail if there are multiple distinct base revisions" in {
+    val ROW = Map("ROW" -> "3", "DATASET" -> "ruimtereis01") ++ profileCSVRow2 ++ Map("BASE_REVISION" -> "9de3f841-0f0d-048b-b3db-4b03ad4834d7") ++ metadataCSVRow2 ++ fileDescriptorCSVRow2 ++ audioVideoCSVRow2
+    val rows = depositCSVRow1 :: ROW :: Nil
```

Comment:
after moving... can you minimize these two lines by looking at the other examples in this test suite?